apps/Learning-Buddy/navigation/AppStack.jsx

```diff
@@ -17,6 +18,7 @@ export const AppStack = () => {
     >
       <Stack.Screen name='Home Screen' component={HomeScreen} />
       <Stack.Screen name='Question Form' component={QuestionForm} />
+      <Stack.Screen name="Sample Test" component={SampleTestScreen} />
```

Comment:
You could rename this to "Answering Screen"; we're going to use this screen and repurpose it with the JSON provided from another screen that navigates here, so no need to call this "sample" or "test".

Reply:
Done 🫠